academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md

```diff
@@ -154,8 +227,19 @@ You can check the build index job progress using Azure Management Portal. Each j
 
 Now that the custom MAKES index files has been generated. You can follow the same procedures as deploying an index from your MAKES subscription, but replacing the **--MakesIndex** parameter with the path to the output of this example.
 
+::: moniker range="makes-1.0"
+
+```cmd
+Kesm.exe DeployHost --HostName "<makes_instance_host_name>" --MakesPackage "https://<MAS_STORAGE_ACCOUNT>.blob.core.windows.net/makes/<YYYY-MM-DD>/"  --MakesHostImageId "<id_from_create_host_resources_command>" --MakesIndex "https://<MAS_STORAGEACCOUNT>.blob.core.windows.net/dev/makes/<YYYY-MM-DD>/microsoft/index/"
```

Comment:
><MAS_STORAGE_ACCOUNT> [](start = 84, length = 21)

why changes needed for makes-1.0?

Reply:
Changing the term "Makes_Storage_Account_Name" to "MAS_STORAGE_ACCOUNT". For makes-3.0/data-share delivery, there's a additional "<MAS_CONTAINER>" param

---
In reply to: [609092229](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/210#discussion_r609092229) [](ancestors = 609092229)